infer/src/integration/SarifReport.ml

```diff
@@ -19,7 +19,18 @@ module ReportSummary = struct
   let pp fmt {n_issues= _; issue_type_counts} =
     let string_of_issue ~issue_type ~issue_type_hum =
       let shortDescription = {Sarifbug_j.text= issue_type_hum} in
-      let rule = {Sarifbug_j.id= issue_type; shortDescription} in
+      let help_uri = 
```

Comment:
Better compute this accurately for all issue types instead. You can get the corresponding IssueType.t from their unique id (which are the strings you have here) using `IssueType.find_from_string`, then generate the URL by exposing `abs_url_of_issue_type` currently in Help.ml so that it's all in one place (that function can stay in Help.ml, you just need to add it to Help.mli):

https://github.com/facebook/infer/blob/9f75f8017f4a9090be4f8057ea06e7118c5ba6ba/infer/src/integration/Help.ml#L24-L25

Reply:
Done. Thanks for pointing this out!